docs/tutorials/neutron-data.ipynb

```diff
@@ -444,7 +443,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "d = sc.neutron.convert(d, Dim.Tof, Dim.DSpacing)"
+    "d = sn.convert(d, Dim.Tof, Dim.DSpacing)"
```

Comment:
I recommend keeping `sc.neutron` in the notebook. Having `sc` and `sn` is too confusing.